### ISSUE TITLE
CPU temp keeps reverting to wifi temp

### ISSUE THREAD

**Issue by lavendercolor** [mgalgs/gnome-shell-system-monitor-applet#63]
after each session login, cpu thermal is reverted to wifi thermal.
OS: Ubuntu 23.10 x86, Kernel: 6.5.0-21-generic, Shell: bash 5.2.15, DE: GNOME 45.2.

example lm-sensors output: 

```
iwlwifi_1-virtual-0
Adapter: Virtual device
temp1:        +37.0°C  

nvme-pci-0300
Adapter: PCI adapter
Composite:    +32.9°C  (low  =  -5.2°C, high = +79.8°C)
                       (crit = +84.8°C)

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 0:        +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 1:        +36.0°C  (high = +100.0°C, crit = +100.0°C)

BAT0-acpi-0
Adapter: ACPI interface
in0:           7.60 V  

```

![image](https://github.com/mgalgs/gnome-shell-system-monitor-applet/assets/143670768/2f718375-898d-4fb5-bde0-be482d101652)

**Comment by ixevix** [comment]
I have the same issue except it reverts to acpitz - 1. Archlinux GNOME 46. System monitor from the https://extensions.gnome.org/extension/3010/system-monitor-next/ website.

**Comment by ZimbiX** [comment]
This looks related: https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet/issues/381

**Comment by mgalgs** [comment]
Can anyone who's experiencing this issue please give the proposed fix (#73) a test drive?

**Comment by ixevix** [comment]
I'll give it a spin. Early next week at the latest.

**Comment by ixevix** [comment]
I tried the rschupp:check_sensors branch but unfortunately I can't edit settings there. 
![image](https://github.com/mgalgs/gnome-shell-system-monitor-applet/assets/820718/ebe28d9e-733e-4726-8059-c229e63498db)

**Comment by ixevix** [comment]
After adding the missing ) on both lines 457 and 515 in prefs.js I managed to open the settings dialog. Will continue testing.

**Comment by mgalgs** [comment]
Fixed-by: #73
(I think -- if someone could please confirm that would be great. I wasn't experiencing this issue before so can't really confirm the fix.)
